bodhi/util.py

```diff
@@ -440,7 +440,13 @@ def update2html(context, update):
     else:
         title = update['title']
 
-    url = request.route_url('update', id=title)
+    alias = None
+    if hasattr(update, 'alias'):
+        alias = update.alias
+    else:
+        alias = update.get('alias')
```

Comment:
So all updates are getting an alias but not necessarily a title? Sounds kinda odd to me